qiskit_machine_learning/kernels/statevector_kernel.py

```diff
@@ -0,0 +1,191 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2022.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+"""Fidelity Statevector Kernel"""
+
+from __future__ import annotations
+
+import numpy as np
+from qiskit import QuantumCircuit
+from qiskit.quantum_info import Statevector
+
+from .base_kernel import BaseKernel
+
+
+class FidelityStatevectorKernel(BaseKernel):
+    r"""
+    A reference implementation of the quantum kernel interface limited to classically simulated
+    statevectors.
+
+    The general task of machine learning is to find and study patterns in data. For many
```

Comment:
This is a nice explanation! Shouldn't this better go into a more general place like the `kernels` init file or so? Hiding general docs into a specific class is makes them harder accessible (and might lead to duplications) 🙂

I would say that starting from 
```
Here, the kernel function is defined as ...
``` 
it really is only class specific 🙂

Reply:
I moved this section of text into the `kernels` init file and removed it from the fidelity kernel docstring.